docs/partners/plan-provisioning-woocommerce.md

```diff
@@ -0,0 +1,10 @@
+# Provisioning Woo Commerce Extensions
+If your Jetpack hosting partner account supports creating WooCommerce Extensions through the Jetpack hosting partner program this document provides additional information.
```

Comment:
I'd probably add a command before "this document provides additional information".